packages/features/auth/lib/verifyEmail.ts

```diff
@@ -54,3 +54,37 @@ export const sendEmailVerification = async ({ email, language, username }: Verif
 
   return { ok: true, skipped: false };
 };
+
+export const sendEmailVerificationByCode = async ({ email, language, username }: VerifyEmailType) => {
+  const token = randomBytes(3).toString("hex");
```

Comment:
we use totp library to generate OTP code. check in create.handler file in packages/trpc/server/routers/viewer/organizations/create.handler.ts

Reply:
will check this out.